### DISCUSSION TITLE
Welcome to React-TodayTalks-v2 Discussions!

### DISCUSSION THREAD

**Discussion by hollynight0** [sr-richard-collins/React-TodayTalks-v2#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hollynight0** [comment]
How can I implement Server-Side rendering in react?

**Reply by hollynight0** [reply]
To implement Server-Side rendering in react, follow these steps

Step 1: Create a new Express.js application. ...
Step 2: Set up the environment configuration. ...
Step 3: Set up the server. ...
Step 4: Create the home page. ...
Step 5: Create the products page. ...
Step 6: Create the API endpoint. ...
Step 7: Start the server.

**Reply by DAWID-NGUYEN-S16** [reply]
> How can I implement Server-Side rendering in react?

We have to change .htaccess file.
